week-2-practise/level-2/2-validation.js

```diff
@@ -4,7 +4,9 @@
 // - is less than or equal to 100
 // Tip: write other small functions for each requirement
 
-function validate(num) {}
+function validate(num) {
+  return (typeof num === "number" && num > 0 && num <= 100)?true:false;
```

Comment:
Nice use a of a ternary operator!

The instructions ask you to write small functions for each required check. An example might be to write function called `isNumber` that checks if the variable is a number.

Reply:
good job